certifier_service/certlib/certlib_support.go

```diff
@@ -1759,6 +1759,18 @@ func GetSubjectNameFromCert(cert *x509.Certificate) *string {
 	return &cert.Subject.CommonName
 }
 
+func GetVcekExtValue(ext pkix.Extension) uint8 {
+	if ext.Value[0] != 0x2 {
+		fmt.Printf("Invalid extension type!\n")
+		return ^uint8(0)
```

Comment:
You are returning a `255` here? Is this what you are expecting? Would it make more sense to to an error return here?

Reply:
I'm treating 0xFF as an error here explicitly. If 0xFF is valid for these SPL values, I'll have to change the design.

Reply:
We may have to change the design here. Even though the field is signed (ASN.1 standard), they zero pad the field, so the overall integer number could still be a max unsigned eight bit integer.

Reply:
Understood. Thanks for the info. I'll update both the C++ and Golang code for that.